test/framework/module_generator.py

```diff
@@ -249,8 +249,8 @@ def test_load_msg(self):
             tcl_load_msg = '\n'.join([
                 '',
                 "if { [ module-info mode load ] } {",
-                "    puts stderr \"test \\$test \\$test",
-                "test \\$foo \\$bar\"",
+                "    puts stderr \"\"\"test \\$test \\$test",
+                "test \\$foo \\$bar\"\"\"",
```

Comment:
It should indeed behave differently on ```\n``` - should i put the to_str method back in?